tests/test_main.py

```diff
@@ -2,13 +2,22 @@
 import threading
 import time
 
+import pytest
 import requests
 
 from uvicorn.config import Config
 from uvicorn.main import Server
 
 
-def test_run():
+@pytest.mark.parametrize(
+    "host, url",
+    [
+        pytest.param(None, "http://127.0.0.1:8000", id="default"),
+        pytest.param("localhost", "http://127.0.0.1:8000", id="hostname"),
+        pytest.param("::", "http://[::]:8000", id="ipv6"),
```

Comment:
```suggestion
        pytest.param("::1", "http://[::1]:8000", id="ipv6"),
```

Reply:
I think that is why the CI doesnt pass on windows, we want to test on loopback

Reply:
Some small context: 
According to https://tools.ietf.org/html/rfc4291 ,
>         ::1                            the loopback address
>         ::                             the unspecified address

I.e. `[::]` is like `0.0.0.0`

Reply:
So I'd recommend to change url to `"http://[::1]:8000"`, but keep host `"::"`